platform/features/listview/src/directives/MCTGesture.js

```diff
@@ -0,0 +1,15 @@
+define(function () {
+    function MCTGesture(gestureService){
+        return {
+            restrict : 'A',
+            link : function($scope, $element, attrs){
+                gestureService.attachGestures(
+                    $element,
+                    $scope.composition.asDomainObject,
```

Comment:
This is tightly coupled to the naming of the object in the list view.  You should use an isolate scope in your directive definition so that this can be specified more clearly:

```javascript
return {
    restrict : 'A',
    scope: {
        domainObject: "=mctObject" // maps attribute on element to scope variable
    },
    link : function($scope, $element, attrs){
        gestureService.attachGestures(
            $element,
            $scope.domainObject, // uses that scope variable
            attrs.mctGesture.split(",")
        )
    }
}
```

Then, in listview.html:

```html
<tr ng-repeat="composition in composees|orderBy:orderByField:reverseSort"
    mct-gesture="menu,info"
    mct-object="composition.asDomainObject">
```